src/main/java/frc/robot/subsystems/ClawJointSubsystem.java

```diff
@@ -0,0 +1,93 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.spark.SparkBase;
+import com.revrobotics.spark.SparkClosedLoopController;
+import com.revrobotics.spark.SparkLowLevel;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.LimitSwitchConfig;
+import com.revrobotics.spark.config.SparkMaxConfig;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+
+
+public class ClawJointSubsystem extends SubsystemBase {
+    private final SparkMax motor;
+    private final AbsoluteEncoder absoluteEncoder;
+    private final DigitalInput limitSwitchLeft;
+    private final DigitalInput limitSwitchRight;
+    private final SparkClosedLoopController controller;
+
+
+    public ClawJointSubsystem(){
+        motor = new SparkMax(RobotMap.CLAWJOINT_MOTOR, SparkLowLevel.MotorType.kBrushless);
+
+        SparkMaxConfig config = new SparkMaxConfig();
```

Comment:
first do all the configuring for the `SparkMax` and only then retrieve the `SparkClosedLoopController` and `AbsoluteEncoder`